types/ssh2/index.d.ts

```diff
@@ -494,6 +494,8 @@ export interface ExecOptions {
 }
 
 export interface ShellOptions {
+    /** An environment to use for the execution of the shell. */
+    env?: any;
```

Comment:
I think this should be

```suggestion
    env: NodeJS.ProcessEnv;
```

Reply:
Fixed.

Reply:
Fixed.